### ISSUE TITLE
ANR com.google.android.gms.auth.api.signin.RevocationBoundService

### ISSUE THREAD

**Issue by ssoher** [playgameservices/play-games-plugin-for-unity#2598]
```

"main" tid=1 Native
"main" prio=5 tid=1 Native
  | group="main" sCount=1 dsCount=0 obj=0x76f3eab8 self=0x7f9c4a2a00
  | sysTid=7392 nice=-10 cgrp=default sched=0/0 handle=0x7fa04dfa98
  | state=S schedstat=( 1650414053 140470958 2454 ) utm=105 stm=60 core=6 HZ=100
  | stack=0x7fd603f000-0x7fd6041000 stackSize=8MB
  | held mutexes=
 
  at com.google.android.gms.games.NativeSdkCallbacks.onResult (Native method)
 
  at com.google.android.gms.common.api.internal.BasePendingResult$CallbackHandler.handleMessage (unavailable:10)
 
  at android.os.Handler.dispatchMessage (Handler.java:105)
 
  at android.os.Looper.loop (Looper.java:156)
 
  at android.app.ActivityThread.main (ActivityThread.java:6623)
 
  at java.lang.reflect.Method.invoke! (Native method)
 
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:942)
 
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:832)
"Jit thread pool worker thread 0" tid=2 Native
"Jit thread pool worker thread 0" prio=5 tid=2 Native (still starting up)
  | group="" sCount=1 dsCount=0 obj=0x0 self=0x7f9400d000
  | sysTid=7397 nice=9 cgrp=default sched=0/0 handle=0x7f9ba0a450
  | state=S schedstat=( 66842085 24469477 51 ) utm=6 stm=0 core=6 HZ=100
  | stack=0x7f9b90c000-0x7f9b90e000 stackSize=1021KB
  | held mutexes=
"ReferenceQueueDaemon" tid=4 Waiting
"ReferenceQueueDaemon" daemon prio=5 tid=4 Waiting
  | group="system" sCount=1 dsCount=0 obj=0x12cbd1f0 self=0x7f9c553400
  | sysTid=7399 nice=0 cgrp=default sched=0/0 handle=0x7f9b80c450
  | state=S schedstat=( 5740470 1043490 21 ) utm=0 stm=0 core=5 HZ=100
  | stack=0x7f9b70a000-0x7f9b70c000 stackSize=1037KB
  | held mutexes=
 
  at java.lang.Object.wait! (Native method)
- waiting on <0x09584cfc> (a java.lang.Class<java.lang.ref.ReferenceQueue>)
 
  at java.lang.Daemons$ReferenceQueueDaemon.run (Daemons.java:150)
- locked <0x09584cfc> (a java.lang.Class<java.lang.ref.ReferenceQueue>)
 
  at java.lang.Thread.run (Thread.java:776)
"FinalizerDaemon" tid=5 Waiting
"FinalizerDaemon" daemon prio=5 tid=5 Waiting
  | group="system" sCount=1 dsCount=0 obj=0x12cbd280 self=0x7f9c553e00
  | sysTid=7400 nice=0 cgrp=default sched=0/0 handle=0x7f9b707450
  | state=S schedstat=( 8758074 10497135 24 ) utm=0 stm=0 core=5 HZ=100
  | stack=0x7f9b605000-0x7f9b607000 stackSize=1037KB
  | held mutexes=
 
  at java.lang.Object.wait! (Native method)
- waiting on <0x08f5bf85> (a java.lang.Object)
 
  at java.lang.Object.wait (Object.java:407)
 
  at java.lang.ref.ReferenceQueue.remove (ReferenceQueue.java:188)
- locked <0x08f5bf85> (a java.lang.Object)
 
  at java.lang.ref.ReferenceQueue.remove (ReferenceQueue.java:209)
 
  at java.lang.Daemons$FinalizerDaemon.run (Daemons.java:204)
 
  at java.lang.Thread.run (Thread.java:776)
"FinalizerWatchdogDaemon" tid=6 Waiting
"FinalizerWatchdogDaemon" daemon prio=5 tid=6 Waiting
  | group="system" sCount=1 dsCount=0 obj=0x12cbd310 self=0x7f9c555c00
  | sysTid=7401 nice=0 cgrp=default sched=0/0 handle=0x7f9b602450
  | state=S schedstat=( 422082 8172084 10 ) utm=0 stm=0 core=4 HZ=100
  | stack=0x7f9b500000-0x7f9b502000 stackSize=1037KB
  | held mutexes=
 
  at java.lang.Object.wait! (Native method)
- waiting on <0x06db41da> (a java.lang.Daemons$FinalizerWatchdogDaemon)
 
  at java.lang.Daemons$FinalizerWatchdogDaemon.sleepUntilNeeded (Daemons.java:269)
- locked <0x06db41da> (a java.lang.Daemons$FinalizerWatchdogDaemon)
 
  at java.lang.Daemons$FinalizerWatchdogDaemon.run (Daemons.java:249)
 
  at java.lang.Thread.run (Thread.java:776)
"Binder:7392_1" tid=8 Native
"Binder:7392_1" prio=5 tid=8 Native
  | group="main" sCount=1 dsCount=0 obj=0x12cbd4c0 self=0x7f94016400
  | sysTid=7403 nice=0 cgrp=default sched=0/0 handle=0x7f9b2fa450
  | state=S schedstat=( 117307448 67968337 456 ) utm=3 stm=8 core=5 HZ=100
  | stack=0x7f9b200000-0x7f9b202000 stackSize=1005KB
  | held mutexes=
"Binder:7392_2" tid=9 Native
"Binder:7392_2" prio=5 tid=9 Native
  | group="main" sCount=1 dsCount=0 obj=0x12cbd550 self=0x7f916e4000
  | sysTid=7404 nice=0 cgrp=default sched=0/0 handle=0x7f9b1f8450
  | state=S schedstat=( 109425073 53188899 444 ) utm=5 stm=5 core=7 HZ=100
  | stack=0x7f9b0fe000-0x7f9b100000 stackSize=1005KB
  | held mutexes=
"Profile Saver" tid=10 Native
"Profile Saver" daemon prio=10 tid=10 Native
  | group="system" sCount=1 dsCount=0 obj=0x12cbfca0 self=0x7f94024800
  | sysTid=7411 nice=-10 cgrp=default sched=0/0 handle=0x7f9b0f3450
  | state=S schedstat=( 11430467 137864 18 ) utm=0 stm=1 core=7 HZ=100
  | stack=0x7f9aff9000-0x7f9affb000 stackSize=1005KB
  | held mutexes=
"Measurement Worker" tid=11 TimedWaiting
"Measurement Worker" prio=5 tid=11 TimedWaiting
  | group="main" sCount=1 dsCount=0 obj=0x12e89700 self=0x7f94053600
  | sysTid=7418 nice=10 cgrp=bg_non_interactive sched=0/0 handle=0x7f716cd450
  | state=S schedstat=( 60926140 35101983 113 ) utm=5 stm=1 core=3 HZ=100
  | stack=0x7f715cb000-0x7f715cd000 stackSize=1037KB
  | held mutexes=
 
  at java.lang.Object.wait! (Native method)
- waiting on <0x095b3ce8> (a java.lang.Object)
 
  at java.lang.Object.wait (Object.java:407)
 
  at il.run (com.google.android.gms.dynamite_measurementdynamite@17455050@17.4.55 (040406-248795830):15)
- locked <0x095b3ce8> (a java.lang.Object)
"pool-1-thread-1" tid=12 TimedWaiting
"pool-1-thread-1" prio=5 tid=12 TimedWaiting
  | group="main" sCount=1 dsCount=0 obj=0x12d7a940 self=0x7f8c87e800
  | sysTid=7417 nice=0 cgrp=default sched=0/0 handle=0x7f9ad06450
  | state=S schedstat=( 51156246 17961304 121 ) utm=5 stm=0 core=2 HZ=100
  | stack=0x7f9ac04000-0x7f9ac06000 stackSize=1037KB
  | held mutexes=
 
  at java.lang.Object.wait! (Native method)
- waiting on <0x0207a60b> (a java.lang.Object)
 
  at java.lang.Thread.parkFor$ (Thread.java:2142)
- locked <0x0207a60b> (a java.lang.Object)
 
  at sun.misc.Unsafe.park (Unsafe.java:325)
 
  at java.util.concurrent.locks.LockSupport.parkNanos (LockSupport.java:201)
 
  at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos (AbstractQueuedSynchronizer.java:2077)
 
  at java.util.concurrent.LinkedBlockingQueue.poll (LinkedBlockingQueue.java:438)
 
  at java.util.concurrent.ThreadPoolExecutor.getTask (ThreadPoolExecutor.java:1057)
 
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1118)
 
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:607)
 
  at java.lang.Thread.run (Thread.java:776)
"UnityMain" tid=13 Native
"UnityMain" prio=5 tid=13 Native
  | group="main" sCount=1 dsCount=0 obj=0x12c8da60 self=0x7f916e1800
  | sysTid=7416 nice=0 cgrp=default sched=0/0 handle=0x7f71c54450
  | state=S schedstat=( 6807992052 1216480575 22514 ) utm=605 stm=75 core=1 HZ=100
  | stack=0x7f71b52000-0x7f71b54000 stackSize=1037KB
  | held mutexes=
 
  at com.unity3d.player.UnityPlayer.nativeRender (Native method)
 
  at com.unity3d.player.UnityPlayer.c (unavailable)
 
  at com.unity3d.player.UnityPlayer$e$2.queueIdle (unavailable)
 
  at android.os.MessageQueue.next (MessageQueue.java:456)
 
  at android.os.Looper.loop (Looper.java:138)
 
  at com.unity3d.player.UnityPlayer$e.run (unavailable)
"pool-2-thread-1" tid=14 Waiting
"pool-2-thread-1" prio=5 tid=14 Waiting
  | group="main" sCount=1 dsCount=0 obj=0x12eff430 self=0x7f6f94d000
  | sysTid=7420 nice=0 cgrp=default sched=0/0 handle=0x7f715c8450
  | state=S schedstat=( 63994111 20581047 155 ) utm=3 stm=3 core=5 HZ=100
  | stack=0x7f714c6000-0x7f714c8000 stackSize=1037KB
  | held mutexes=
 
  at java.lang.Object.wait! (Native method)
- waiting on <0x0bd7cf01> (a java.lang.Object)
 
  at java.lang.Thread.parkFor$ (Thread.java:2142)
- locked <0x0bd7cf01> (a java.lang.Object)
 
  at sun.misc.Unsafe.park (Unsafe.java:325)
 
  at java.util.concurrent.locks.LockSupport.park (LockSupport.java:161)
 
  at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await (AbstractQueuedSynchronizer.java:2035)
 
  at java.util.concurrent.LinkedBlockingQueue.take (LinkedBlockingQueue.java:413)
 
  at java.util.concurrent.ThreadPoolExecutor.getTask (ThreadPoolExecutor.java:1058)
 
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1118)
 
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:607)
 
  at java.lang.Thread.run (Thread.java:776)
"RenderThread" tid=15 Native
"RenderThread" prio=5 tid=15 Native
  | group="main" sCount=1 dsCount=0 obj=0x134bc5e0 self=0x7f67361a00
  | sysTid=7513 nice=-10 cgrp=default sched=0/0 handle=0x7f6a17a450
  | state=S schedstat=( 43877186 12912289 58 ) utm=2 stm=2 core=4 HZ=100
  | stack=0x7f6a080000-0x7f6a082000 stackSize=1005KB
  | held mutexes=
"Binder:7392_3" tid=16 Native
"Binder:7392_3" prio=5 tid=16 Native
  | group="main" sCount=1 dsCount=0 obj=0x134bcd30 self=0x7f59135a00
  | sysTid=7540 nice=0 cgrp=default sched=0/0 handle=0x7f69c9b450
  | state=S schedstat=( 77263904 73019535 304 ) utm=2 stm=5 core=5 HZ=100
  | stack=0x7f69ba1000-0x7f69ba3000 stackSize=1005KB
  | held mutexes=
"PlatformServiceBridgeHandlerThread" tid=17 Native
"PlatformServiceBridgeHandlerThread" prio=5 tid=17 Native
  | group="main" sCount=1 dsCount=0 obj=0x12e6ddc0 self=0x7f66922c00
  | sysTid=7486 nice=0 cgrp=default sched=0/0 handle=0x7f626d1450
  | state=S schedstat=( 11950321 7829006 38 ) utm=0 stm=1 core=4 HZ=100
  | stack=0x7f625cf000-0x7f625d1000 stackSize=1037KB
  | held mutexes=
 
  at android.os.MessageQueue.nativePollOnce (Native method)
 
  at android.os.MessageQueue.next (MessageQueue.java:356)
 
  at android.os.Looper.loop (Looper.java:138)
 
  at android.os.HandlerThread.run (HandlerThread.java:61)
"CrAsyncTask #2" tid=18 TimedWaiting
"CrAsyncTask #2" prio=5 tid=18 TimedWaiting
  | group="main" sCount=1 dsCount=0 obj=0x12dad550 self=0x7f66664e00
  | sysTid=7475 nice=10 cgrp=default sched=0/0 handle=0x7f661f5450
  | state=S schedstat=( 6303488 1857657 8 ) utm=0 stm=0 core=5 HZ=100
  | stack=0x7f660f3000-0x7f660f5000 stackSize=1037KB
  | held mutexes=
 
  at java.lang.Object.wait! (Native method)
- waiting on <0x061b3f94> (a java.lang.Object)
 
  at java.lang.Thread.parkFor$ (Thread.java:2142)
- locked <0x061b3f94> (a java.lang.Object)
 
  at sun.misc.Unsafe.park (Unsafe.java:325)
 
  at java.util.concurrent.locks.LockSupport.parkNanos (LockSupport.java:201)
 
  at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos (AbstractQueuedSynchronizer.java:2077)
 
  at java.util.concurrent.ArrayBlockingQueue.poll (ArrayBlockingQueue.java:373)
 
  at java.util.concurrent.ThreadPoolExecutor.getTask (ThreadPoolExecutor.java:1057)
 
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1118)
 
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:607)
 
  at aNn.run (unavailable:3)
 
  at java.lang.Thread.run (Thread.java:776)
"GoogleApiHandler" tid=19 Native
"GoogleApiHandler" prio=5 tid=19 Native
  | group="main" sCount=1 dsCount=0 obj=0x1334eaf0 self=0x7f66448a00
  | sysTid=7504 nice=9 cgrp=default sched=0/0 handle=0x7f587ff450
  | state=S schedstat=( 14589270 4712238 26 ) utm=1 stm=0 core=5 HZ=100
  | stack=0x7f586fd000-0x7f586ff000 stackSize=1037KB
  | held mutexes=
 
  at android.os.MessageQueue.nativePollOnce (Native method)
 
  at android.os.MessageQueue.next (MessageQueue.java:356)
 
  at android.os.Looper.loop (Looper.java:138)
 
  at android.os.HandlerThread.run (HandlerThread.java:61)
"AudioThread" tid=20 Native
"AudioThread" prio=5 tid=20 Native
  | group="main" sCount=1 dsCount=0 obj=0x131e6700 self=0x7f639caa00
  | sysTid=7481 nice=0 cgrp=default sched=0/0 handle=0x7f62bc2450
  | state=S schedstat=( 9895577 3613800 34 ) utm=0 stm=0 core=4 HZ=100
  | stack=0x7f62ac8000-0x7f62aca000 stackSize=1005KB
  | held mutexes=
"UnityChoreographer" tid=21 Native
"UnityChoreographer" prio=5 tid=21 Native
  | group="main" sCount=1 dsCount=0 obj=0x12cb5160 self=0x7f68e7bc00
  | sysTid=7463 nice=-2 cgrp=default sched=0/0 handle=0x7f66b40450
  | state=S schedstat=( 557525726 244847487 1080 ) utm=42 stm=13 core=5 HZ=100
  | stack=0x7f66a3e000-0x7f66a40000 stackSize=1037KB
  | held mutexes=
 
  at android.os.MessageQueue.nativePollOnce (Native method)
 
  at android.os.MessageQueue.next (MessageQueue.java:356)
 
  at android.os.Looper.loop (Looper.java:138)
 
  at android.os.HandlerThread.run (HandlerThread.java:61)
"Chrome_IOThread" tid=22 Native
"Chrome_IOThread" prio=7 tid=22 Native
  | group="main" sCount=1 dsCount=0 obj=0x12e5d820 self=0x7f6387de00
  | sysTid=7479 nice=-4 cgrp=default sched=0/0 handle=0x7f62dbc450
  | state=S schedstat=( 165973240 46938536 1272 ) utm=11 stm=5 core=4 HZ=100
  | stack=0x7f62cc2000-0x7f62cc4000 stackSize=1005KB
  | held mutexes=
"OkHttp ConnectionPool" tid=23 TimedWaiting
"OkHttp ConnectionPool" daemon prio=4 tid=23 TimedWaiting
  | group="main" sCount=1 dsCount=0 obj=0x12cbdca0 self=0x7f6801fe00
  | sysTid=7465 nice=10 cgrp=default sched=0/0 handle=0x7f663ff450
  | state=S schedstat=( 5946096 10180463 44 ) utm=0 stm=0 core=6 HZ=100
  | stack=0x7f662fd000-0x7f662ff000 stackSize=1037KB
  | held mutexes=
 
  at java.lang.Object.wait! (Native method)
- waiting on <0x07bba1a6> (a com.android.okhttp.ConnectionPool)
 
  at com.android.okhttp.ConnectionPool.performCleanup (ConnectionPool.java:305)
- locked <0x07bba1a6> (a com.android.okhttp.ConnectionPool)
 
  at com.android.okhttp.ConnectionPool.runCleanupUntilPoolIsEmpty (ConnectionPool.java:242)
 
  at com.android.okhttp.ConnectionPool.-wrap0 (ConnectionPool.java)
 
  at com.android.okhttp.ConnectionPool$1.run (ConnectionPool.java:97)
 
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1133)
 
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:607)
 
  at java.lang.Thread.run (Thread.java:776)
"GoogleApiHandler" tid=25 Native
"GoogleApiHandler" prio=5 tid=25 Native
  | group="main" sCount=1 dsCount=0 obj=0x12df7af0 self=0x7f681a4a00
  | sysTid=7476 nice=9 cgrp=default sched=0/0 handle=0x7f637ff450
  | state=S schedstat=( 10144791 20908023 39 ) utm=0 stm=1 core=5 HZ=100
  | stack=0x7f636fd000-0x7f636ff000 stackSize=1037KB
  | held mutexes=
 
  at android.os.MessageQueue.nativePollOnce (Native method)
 
  at android.os.MessageQueue.next (MessageQueue.java:356)
 
  at android.os.Looper.loop (Looper.java:138)
 
  at android.os.HandlerThread.run (HandlerThread.java:61)
"CrAsyncTask #1" tid=27 TimedWaiting
"CrAsyncTask #1" prio=5 tid=27 TimedWaiting
  | group="main" sCount=1 dsCount=0 obj=0x12cbb940 self=0x7f8c9b9800
  | sysTid=7472 nice=10 cgrp=default sched=0/0 handle=0x7f63bfb450
  | state=S schedstat=( 2885990 614374 11 ) utm=0 stm=0 core=5 HZ=100
  | stack=0x7f63af9000-0x7f63afb000 stackSize=1037KB
  | held mutexes=
 
  at java.lang.Object.wait! (Native method)
- waiting on <0x028897e7> (a java.lang.Object)
 
  at java.lang.Thread.parkFor$ (Thread.java:2142)
- locked <0x028897e7> (a java.lang.Object)
 
  at sun.misc.Unsafe.park (Unsafe.java:325)
 
  at java.util.concurrent.locks.LockSupport.parkNanos (LockSupport.java:201)
 
  at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos (AbstractQueuedSynchronizer.java:2077)
 
  at java.util.concurrent.ArrayBlockingQueue.poll (ArrayBlockingQueue.java:373)
 
  at java.util.concurrent.ThreadPoolExecutor.getTask (ThreadPoolExecutor.java:1057)
 
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1118)
 
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:607)
 
  at aNn.run (unavailable:3)
 
  at java.lang.Thread.run (Thread.java:776)
"TaskSchedulerFo" tid=28 Native
"TaskSchedulerFo" prio=5 tid=28 Native
  | group="main" sCount=1 dsCount=0 obj=0x12e7dee0 self=0x7f69261000
  | sysTid=7478 nice=0 cgrp=default sched=0/0 handle=0x7f62eb9450
  | state=S schedstat=( 81001255 32061718 613 ) utm=4 stm=4 core=7 HZ=100
  | stack=0x7f62dbf000-0x7f62dc1000 stackSize=1005KB
  | held mutexes=
"GAC_Executor[0]" tid=29 Waiting
"GAC_Executor[0]" prio=5 tid=29 Waiting
  | group="main" sCount=1 dsCount=0 obj=0x12e7da60 self=0x7f63916200
  | sysTid=7487 nice=0 cgrp=default sched=0/0 handle=0x7f625cc450
  | state=S schedstat=( 6089899 7564736 21 ) utm=0 stm=0 core=4 HZ=100
  | stack=0x7f624ca000-0x7f624cc000 stackSize=1037KB
  | held mutexes=
 
  at java.lang.Object.wait! (Native method)
- waiting on <0x07715a3d> (a java.lang.Object)
 
  at java.lang.Thread.parkFor$ (Thread.java:2142)
- locked <0x07715a3d> (a java.lang.Object)
 
  at sun.misc.Unsafe.park (Unsafe.java:325)
 
  at java.util.concurrent.locks.LockSupport.park (LockSupport.java:161)
 
  at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await (AbstractQueuedSynchronizer.java:2035)
 
  at java.util.concurrent.LinkedBlockingQueue.take (LinkedBlockingQueue.java:413)
 
  at java.util.concurrent.ThreadPoolExecutor.getTask (ThreadPoolExecutor.java:1058)
 
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1118)
 
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:607)
 
  at Oe.run (PG:5)
 
  at java.lang.Thread.run (Thread.java:776)
"ConnectivityManager" tid=30 Native
"ConnectivityManager" prio=5 tid=30 Native
  | group="main" sCount=1 dsCount=0 obj=0x12ec5f70 self=0x7f8c9b8e00
  | sysTid=7488 nice=0 cgrp=default sched=0/0 handle=0x7f624c7450
  | state=S schedstat=( 5076303 3122395 18 ) utm=0 stm=0 core=7 HZ=100
  | stack=0x7f623c5000-0x7f623c7000 stackSize=1037KB
  | held mutexes=
 
  at android.os.MessageQueue.nativePollOnce (Native method)
 
  at android.os.MessageQueue.next (MessageQueue.java:356)
 
  at android.os.Looper.loop (Looper.java:138)
 
  at android.os.HandlerThread.run (HandlerThread.java:61)
"CleanupReference" tid=31 Waiting
"CleanupReference" daemon prio=5 tid=31 Waiting
  | group="main" sCount=1 dsCount=0 obj=0x12ec5c10 self=0x7f69267400
  | sysTid=7489 nice=0 cgrp=default sched=0/0 handle=0x7f623c2450
  | state=S schedstat=( 238176 37813 3 ) utm=0 stm=0 core=4 HZ=100
  | stack=0x7f622c0000-0x7f622c2000 stackSize=1037KB
  | held mutexes=
 
  at java.lang.Object.wait! (Native method)
- waiting on <0x0cf03232> (a java.lang.Object)
 
  at java.lang.Object.wait (Object.java:407)
 
  at java.lang.ref.ReferenceQueue.remove (ReferenceQueue.java:188)
- locked <0x0cf03232> (a java.lang.Object)
 
  at java.lang.ref.ReferenceQueue.remove (ReferenceQueue.java:209)
 
  at aJg.run (PG:3)
"Chrome_InProcRendererThread" tid=32 Native
"Chrome_InProcRendererThread" prio=5 tid=32 Native
  | group="main" sCount=1 dsCount=0 obj=0x131e6820 self=0x7f639eae00
  | sysTid=7490 nice=0 cgrp=default sched=0/0 handle=0x7f6229d450
  | state=S schedstat=( 323095357 52415517 720 ) utm=27 stm=5 core=4 HZ=100
  | stack=0x7f621a3000-0x7f621a5000 stackSize=1005KB
  | held mutexes=
"Chrome_InProcGpuThread" tid=33 Native
"Chrome_InProcGpuThread" prio=7 tid=33 Native
  | group="main" sCount=1 dsCount=0 obj=0x131e68b0 self=0x7f6926a600
  | sysTid=7492 nice=-4 cgrp=default sched=0/0 handle=0x7f620a3450
  | state=S schedstat=( 160299263 31330674 402 ) utm=13 stm=3 core=1 HZ=100
  | stack=0x7f61fa9000-0x7f61fab000 stackSize=1005KB
  | held mutexes=
"TaskSchedulerFo" tid=34 Native
"TaskSchedulerFo" prio=5 tid=34 Native
  | group="main" sCount=1 dsCount=0 obj=0x1323c160 self=0x7f67195e00
  | sysTid=7480 nice=0 cgrp=default sched=0/0 handle=0x7f62cbf450
  | state=S schedstat=( 35257136 33529011 132 ) utm=2 stm=1 core=4 HZ=100
  | stack=0x7f62bc5000-0x7f62bc7000 stackSize=1005KB
  | held mutexes=
"GAC_Executor[1]" tid=35 Waiting
"GAC_Executor[1]" prio=5 tid=35 Waiting
  | group="main" sCount=1 dsCount=0 obj=0x1323cee0 self=0x7f69268800
  | sysTid=7502 nice=0 cgrp=default sched=0/0 handle=0x7f58ebf450
  | state=S schedstat=( 1507811 18151928 15 ) utm=0 stm=0 core=4 HZ=100
  | stack=0x7f58dbd000-0x7f58dbf000 stackSize=1037KB
  | held mutexes=
 
  at java.lang.Object.wait! (Native method)
- waiting on <0x063ccb83> (a java.lang.Object)
 
  at java.lang.Thread.parkFor$ (Thread.java:2142)
- locked <0x063ccb83> (a java.lang.Object)
 
  at sun.misc.Unsafe.park (Unsafe.java:325)
 
  at java.util.concurrent.locks.LockSupport.park (LockSupport.java:161)
 
  at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await (AbstractQueuedSynchronizer.java:2035)
 
  at java.util.concurrent.LinkedBlockingQueue.take (LinkedBlockingQueue.java:413)
 
  at java.util.concurrent.ThreadPoolExecutor.getTask (ThreadPoolExecutor.java:1058)
 
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1118)
 
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:607)
 
  at Oe.run (PG:5)
 
  at java.lang.Thread.run (Thread.java:776)
"pool-3-thread-1" tid=36 Waiting
"pool-3-thread-1" prio=5 tid=36 Waiting
  | group="main" sCount=1 dsCount=0 obj=0x13385ee0 self=0x7f672ab400
  | sysTid=7506 nice=0 cgrp=default sched=0/0 handle=0x7f583ff450
  | state=S schedstat=( 21447190 14736822 51 ) utm=2 stm=0 core=5 HZ=100
  | stack=0x7f582fd000-0x7f582ff000 stackSize=1037KB
  | held mutexes=
 
  at java.lang.Object.wait! (Native method)
- waiting on <0x0836c100> (a java.lang.Object)
 
  at java.lang.Thread.parkFor$ (Thread.java:2142)
- locked <0x0836c100> (a java.lang.Object)
 
  at sun.misc.Unsafe.park (Unsafe.java:325)
 
  at java.util.concurrent.locks.LockSupport.park (LockSupport.java:161)
 
  at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await (AbstractQueuedSynchronizer.java:2035)
 
  at java.util.concurrent.LinkedBlockingQueue.take (LinkedBlockingQueue.java:413)
 
  at java.util.concurrent.ThreadPoolExecutor.getTask (ThreadPoolExecutor.java:1058)
 
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1118)
 
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:607)
 
  at java.lang.Thread.run (Thread.java:776)
"JavaBridge" tid=37 Native
"JavaBridge" prio=7 tid=37 Native
  | group="main" sCount=1 dsCount=0 obj=0x133da1f0 self=0x7f61d7ce00
  | sysTid=7507 nice=0 cgrp=default sched=0/0 handle=0x7f582fa450
  | state=S schedstat=( 3573594 6543490 10 ) utm=0 stm=0 core=5 HZ=100
  | stack=0x7f581f8000-0x7f581fa000 stackSize=1037KB
  | held mutexes=
 
  at android.os.MessageQueue.nativePollOnce (Native method)
 
  at android.os.MessageQueue.next (MessageQueue.java:356)
 
  at android.os.Looper.loop (Looper.java:138)
 
  at android.os.HandlerThread.run (HandlerThread.java:61)
"ModernAsyncTask #1" tid=38 Waiting
"ModernAsyncTask #1" prio=5 tid=38 Waiting
  | group="main" sCount=1 dsCount=0 obj=0x1351d1f0 self=0x7f6644b200
  | sysTid=7621 nice=10 cgrp=bg_non_interactive sched=0/0 handle=0x7f69b7e450
  | state=S schedstat=( 1005469 992135 1 ) utm=0 stm=0 core=6 HZ=100
  | stack=0x7f69a7c000-0x7f69a7e000 stackSize=1037KB
  | held mutexes=
 
  at java.lang.Object.wait! (Native method)
- waiting on <0x041e5d39> (a java.lang.Object)
 
  at java.lang.Thread.parkFor$ (Thread.java:2142)
- locked <0x041e5d39> (a java.lang.Object)
 
  at sun.misc.Unsafe.park (Unsafe.java:325)
 
  at java.util.concurrent.locks.LockSupport.park (LockSupport.java:161)
 
  at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await (AbstractQueuedSynchronizer.java:2035)
 
  at java.util.concurrent.LinkedBlockingQueue.take (LinkedBlockingQueue.java:413)
 
  at java.util.concurrent.ThreadPoolExecutor.getTask (ThreadPoolExecutor.java:1058)
 
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1118)
 
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:607)
 
  at java.lang.Thread.run (Thread.java:776)
"TaskSchedulerFo" tid=39 Native
"TaskSchedulerFo" prio=5 tid=39 Native
  | group="main" sCount=1 dsCount=0 obj=0x133daee0 self=0x7f639c5a00
  | sysTid=7483 nice=0 cgrp=default sched=0/0 handle=0x7f629c8450
  | state=S schedstat=( 207919379 79932701 912 ) utm=4 stm=16 core=4 HZ=100
  | stack=0x7f628ce000-0x7f628d0000 stackSize=1005KB
  | held mutexes=
"GAC_Executor[0]" tid=41 Waiting
"GAC_Executor[0]" prio=5 tid=41 Waiting
  | group="main" sCount=1 dsCount=0 obj=0x12cb5af0 self=0x7f6a314c00
  | sysTid=7636 nice=0 cgrp=default sched=0/0 handle=0x7f581f5450
  | state=S schedstat=( 6244999 9604271 15 ) utm=0 stm=0 core=6 HZ=100
  | stack=0x7f580f3000-0x7f580f5000 stackSize=1037KB
  | held mutexes=
 
  at java.lang.Object.wait! (Native method)
- waiting on <0x046b3f7e> (a java.lang.Object)
 
  at java.lang.Thread.parkFor$ (Thread.java:2142)
- locked <0x046b3f7e> (a java.lang.Object)
 
  at sun.misc.Unsafe.park (Unsafe.java:325)
 
  at java.util.concurrent.locks.LockSupport.park (LockSupport.java:161)
 
  at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await (AbstractQueuedSynchronizer.java:2035)
 
  at java.util.concurrent.LinkedBlockingQueue.take (LinkedBlockingQueue.java:413)
 
  at java.util.concurrent.ThreadPoolExecutor.getTask (ThreadPoolExecutor.java:1058)
 
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1118)
 
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:607)
 
  at com.google.android.gms.common.util.concurrent.zza.run (unavailable:6)
 
  at java.lang.Thread.run (Thread.java:776)
"GAC_Executor[1]" tid=42 Waiting
"GAC_Executor[1]" prio=5 tid=42 Waiting
  | group="main" sCount=1 dsCount=0 obj=0x12cb91f0 self=0x7f57f64c00
  | sysTid=7638 nice=0 cgrp=default sched=0/0 handle=0x7f57dff450
  | state=S schedstat=( 1909324 2581979 3 ) utm=0 stm=0 core=1 HZ=100
  | stack=0x7f57cfd000-0x7f57cff000 stackSize=1037KB
  | held mutexes=
 
  at java.lang.Object.wait! (Native method)
- waiting on <0x08769cdf> (a java.lang.Object)
 
  at java.lang.Thread.parkFor$ (Thread.java:2142)
- locked <0x08769cdf> (a java.lang.Object)
 
  at sun.misc.Unsafe.park (Unsafe.java:325)
 
  at java.util.concurrent.locks.LockSupport.park (LockSupport.java:161)
 
  at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await (AbstractQueuedSynchronizer.java:2035)
 
  at java.util.concurrent.LinkedBlockingQueue.take (LinkedBlockingQueue.java:413)
 
  at java.util.concurrent.ThreadPoolExecutor.getTask (ThreadPoolExecutor.java:1058)
 
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1118)
 
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:607)
 
  at com.google.android.gms.common.util.concurrent.zza.run (unavailable:6)
 
  at java.lang.Thread.run (Thread.java:776)
"FMODAudioDevice" tid=43 Native
"FMODAudioDevice" prio=10 tid=43 Native
  | group="main" sCount=1 dsCount=0 obj=0x12cb9700 self=0x7f69269200
  | sysTid=7639 nice=-8 cgrp=default sched=0/0 handle=0x7f57cfa450
  | state=S schedstat=( 393344948 19986360 353 ) utm=0 stm=39 core=1 HZ=100
  | stack=0x7f57bf8000-0x7f57bfa000 stackSize=1037KB
  | held mutexes=
 
  at android.media.AudioTrack.native_write_byte (Native method)
 
  at android.media.AudioTrack.write (AudioTrack.java:1954)
 
  at android.media.AudioTrack.write (AudioTrack.java:1891)
 
  at org.fmod.FMODAudioDevice.run (unavailable)
 
  at java.lang.Thread.run (Thread.java:776)
"Binder:7392_4" tid=45 Native
"Binder:7392_4" prio=5 tid=45 Native
  | group="main" sCount=1 dsCount=0 obj=0x12dadd30 self=0x7f57398e00
  | sysTid=7657 nice=0 cgrp=default sched=0/0 handle=0x7f563d7450
  | state=S schedstat=( 64332456 52693591 259 ) utm=3 stm=3 core=5 HZ=100
  | stack=0x7f562dd000-0x7f562df000 stackSize=1005KB
  | held mutexes=
"AudioTrack" tid=46 Native
"AudioTrack" prio=10 tid=46 Native
  | group="main" sCount=1 dsCount=0 obj=0x12cb9dc0 self=0x7f5848de00
  | sysTid=7640 nice=-16 cgrp=default sched=0/0 handle=0x7f57bf5450
  | state=S schedstat=( 575364 750157 5 ) utm=0 stm=0 core=1 HZ=100
  | stack=0x7f57afb000-0x7f57afd000 stackSize=1005KB
  | held mutexes=
"LooperProvider" tid=47 Native
"LooperProvider" prio=5 tid=47 Native
  | group="main" sCount=1 dsCount=0 obj=0x12d6a5e0 self=0x7f57f65600
  | sysTid=7643 nice=0 cgrp=default sched=0/0 handle=0x7f576f8450
  | state=S schedstat=( 10710729 22451513 39 ) utm=0 stm=1 core=6 HZ=100
  | stack=0x7f575f6000-0x7f575f8000 stackSize=1037KB
  | held mutexes=
 
  at android.os.MessageQueue.nativePollOnce (Native method)
 
  at android.os.MessageQueue.next (MessageQueue.java:356)
 
  at android.os.Looper.loop (Looper.java:138)
 
  at android.os.HandlerThread.run (HandlerThread.java:61)
"gads-1" tid=48 TimedWaiting
"gads-1" prio=5 tid=48 TimedWaiting
  | group="main" sCount=1 dsCount=0 obj=0x12d6a550 self=0x7f66665800
  | sysTid=7644 nice=0 cgrp=default sched=0/0 handle=0x7f575f3450
  | state=S schedstat=( 5285732 15923542 50 ) utm=0 stm=0 core=6 HZ=100
  | stack=0x7f574f1000-0x7f574f3000 stackSize=1037KB
  | held mutexes=
 
  at java.lang.Object.wait! (Native method)
- waiting on <0x01c6ed2c> (a java.lang.Object)
 
  at java.lang.Thread.parkFor$ (Thread.java:2142)
- locked <0x01c6ed2c> (a java.lang.Object)
 
  at sun.misc.Unsafe.park (Unsafe.java:325)
 
  at java.util.concurrent.locks.LockSupport.parkNanos (LockSupport.java:201)
 
  at java.util.concurrent.SynchronousQueue$TransferStack.awaitFulfill (SynchronousQueue.java:432)
 
  at java.util.concurrent.SynchronousQueue$TransferStack.transfer (SynchronousQueue.java:333)
 
  at java.util.concurrent.SynchronousQueue.poll (SynchronousQueue.java:908)
 
  at java.util.concurrent.ThreadPoolExecutor.getTask (ThreadPoolExecutor.java:1057)
 
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1118)
 
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:607)
 
  at java.lang.Thread.run (Thread.java:776)
"AdWorker(Default) #3" tid=49 Waiting
"AdWorker(Default) #3" prio=5 tid=49 Waiting
  | group="main" sCount=1 dsCount=0 obj=0x12d7adc0 self=0x7f672d7000
  | sysTid=7645 nice=0 cgrp=default sched=0/0 handle=0x7f56afa450
  | state=S schedstat=( 6982861 21747400 61 ) utm=0 stm=0 core=6 HZ=100
  | stack=0x7f569f8000-0x7f569fa000 stackSize=1037KB
  | held mutexes=
 
  at java.lang.Object.wait! (Native method)
- waiting on <0x09fe93f5> (a java.lang.Object)
 
  at java.lang.Thread.parkFor$ (Thread.java:2142)
- locked <0x09fe93f5> (a java.lang.Object)
 
  at sun.misc.Unsafe.park (Unsafe.java:325)
 
  at java.util.concurrent.locks.LockSupport.park (LockSupport.java:161)
 
  at java.util.concurrent.SynchronousQueue$TransferStack.awaitFulfill (SynchronousQueue.java:430)
 
  at java.util.concurrent.SynchronousQueue$TransferStack.transfer (SynchronousQueue.java:333)
 
  at java.util.concurrent.SynchronousQueue.take (SynchronousQueue.java:891)
 
  at java.util.concurrent.ThreadPoolExecutor.getTask (ThreadPoolExecutor.java:1058)
 
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1118)
 
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:607)
 
  at java.lang.Thread.run (Thread.java:776)
"AdWorker(NG) #1" tid=59 TimedWaiting
"AdWorker(NG) #1" prio=5 tid=59 TimedWaiting
  | group="main" sCount=1 dsCount=0 obj=0x12df7790 self=0x7f5737f800
  | sysTid=7663 nice=0 cgrp=default sched=0/0 handle=0x7f565e1450
  | state=# schedstat=( 1082397 3454009 9 ) utm=5 stm=0 core=-1 HZ=100
  | stack=0x7f564df000-0x7f564e1000 stackSize=1037KB
  | held mutexes=
 
  at java.lang.Object.wait! (Native method)
- waiting on <0x06f7d58a> (a java.lang.Object)
 
  at java.lang.Thread.parkFor$ (Thread.java:2142)
- locked <0x06f7d58a> (a java.lang.Object)
 
  at sun.misc.Unsafe.park (Unsafe.java:325)
 
  at java.util.concurrent.locks.LockSupport.parkNanos (LockSupport.java:201)
 
  at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos (AbstractQueuedSynchronizer.java:2077)
 
  at java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take (ScheduledThreadPoolExecutor.java:1103)
 
  at java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take (ScheduledThreadPoolExecutor.java:1084)
 
  at java.util.concurrent.ThreadPoolExecutor.getTask (ThreadPoolExecutor.java:1058)
 
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1118)
 
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:607)
 
  at java.lang.Thread.run (Thread.java:776)
"Signal Catcher" tid=3 Runnable
"Signal Catcher" daemon prio=5 tid=3 Runnable
  | group="system" sCount=0 dsCount=0 obj=0x12cbd160 self=0x7f9c4a3400
  | sysTid=7398 nice=0 cgrp=default sched=0/0 handle=0x7f9b909450
  | state=R schedstat=( 77527501 5078853 27 ) utm=3 stm=4 core=6 HZ=100
  | stack=0x7f9b80f000-0x7f9b811000 stackSize=1005KB
  | held mutexes= "mutator lock"(shared held)
"HeapTaskDaemon" tid=7 Blocked
"HeapTaskDaemon" daemon prio=5 tid=7 Blocked
  | group="system" sCount=1 dsCount=0 obj=0x12cbd3a0 self=0x7f9c556600
  | sysTid=7402 nice=0 cgrp=default sched=0/0 handle=0x7f9b4fd450
  | state=S schedstat=( 222272750 43259323 256 ) utm=19 stm=3 core=3 HZ=100
  | stack=0x7f9b3fb000-0x7f9b3fd000 stackSize=1037KB
  | held mutexes=
 
  at dalvik.system.VMRuntime.runHeapTasks (Native method)
- waiting to lock an unknown object
 
  at java.lang.Daemons$HeapTaskDaemon.run (Daemons.java:433)
 
  at java.lang.Thread.run (Thread.java:776)
```

**Comment by CaringSouls** [comment]
Anyone find the solution?
UnityMain" prio=5 tid=15 Native
| group="main" sCount=1 dsCount=0 flags=1 obj=0x12d427f0 self=0x7996511e00
| sysTid=30857 nice=0 cgrp=default sched=0/0 handle=0x7988de04f0
| state=S schedstat=( 37682227386 3948408436 53200 ) utm=3346 stm=422 core=1 HZ=100
| stack=0x7988cde000-0x7988ce0000 stackSize=1037KB
| held mutexes=
#00 pc 000000000001d4ec /system/lib64/libc.so (syscall+28)
#01 pc 00000000000e7040 /system/lib64/libart.so (art::ConditionVariable::WaitHoldingLocks(art::Thread*)+152)
#02 pc 0000000000342e00 /system/lib64/libart.so (art::JNI::PushLocalFrame(_JNIEnv*, int)+476)
#03 pc 000000000087f158 /data/app/com.goxo.doctor.heart.surgery.game-6YmRZiLTCA26Jy8UT5hFTA==/lib/arm64/libunity.so (???)
at com.unity3d.player.UnityPlayer.nativeRender (Native method)
at com.unity3d.player.UnityPlayer.c (unavailable)
at com.unity3d.player.UnityPlayer$e$2.queueIdle (unavailable)
at android.os.MessageQueue.next (MessageQueue.java:402)
at android.os.Looper.loop (Looper.java:150)
at com.unity3d.player.UnityPlayer$e.run (unavailable)

**Comment by vaibhavgpt-google** [comment]
Closing this issue as part of a cleanup of older tickets. Google Play Games v1 is now deprecated. Please upgrade to version v2.1.0 or newer: https://github.com/playgameservices/play-games-plugin-for-unity/releases/tag/v2.1.0. If this issue persists on the new v2 architecture, please open a new ticket.
